actionview/lib/action_view/template/error.rb

```diff
@@ -122,15 +122,11 @@ def source_location
           end + file_name
         end
 
-        def formatted_code_for(source_code, line_counter, indent, output)
-          start_value = (output == :html) ? {} : []
-          source_code.inject(start_value) do |result, line|
+        def formatted_code_for(source_code, line_counter, indent)
+          indent_template = "%#{indent}s: %s"
```

Comment:
Is this an `indent_template` or a `line_template`?

Reply:
The line of the template is being indented by a `indent` amount, so I went with indent template as a better name for what this code is doing, but I'm happy to change it to `line_template` if you feel that it's better.